src/main/java/com/dawkastresu/medical_clinic/UserController.java

```diff
@@ -0,0 +1,42 @@
+package com.dawkastresu.medical_clinic;
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/patients/")
+public class UserController {
+
+    private final PatientRepository patientRepository;
+    private final PatientService patientService;
+
+    @GetMapping
+    public List<Patient> getAllPatients() {
+        return patientRepository.findAll();
+    }
+
+    @GetMapping("/{email}")
+    public Optional<Patient> getPatientByMail(@PathVariable String email) {
```

Comment:
Czy zwracanie optionali w Controllerze ma sens. 
Serwis powinien dostarczać to co Kontroller ma juz zwrocic.